src/core_modules/capture-core/components/CardList/CardList.selectors.js

```diff
@@ -1,23 +1,23 @@
 // @flow
 import { createSelector } from 'reselect';
-import { chunk } from 'capture-core-utils';
 import { elementTypes } from '../../metaData/DataElement';
 
 
-const elementsSelector = props => props.dataElements;
+const elementsSelector = dataElements => dataElements;
 
 // $FlowFixMe[missing-annot] automated comment
 const makeElementsContainerSelector = () => createSelector(
     elementsSelector,
     (elements) => {
-        const newElements = [...elements];
         // $FlowFixMe[prop-missing] automated comment
         const imageDataElement = elements.find(a => a.type === elementTypes.IMAGE);
         if (imageDataElement) {
+            const newElements = [...elements];
             newElements.splice(newElements.indexOf(imageDataElement), 1);
         }
+
         return {
-            dataElementChunks: chunk(elements, 5),
```

Comment:
here we used to chunk the tracked entity attributes. So that we can have them displayed in columns of 5.  

After the discussion [here](https://jira.dhis2.org/browse/DHIS2-9239) we said we wouldnt like it to have in columns